deploy/helm/README.md

```diff
@@ -182,6 +182,7 @@ Keeps security report resources updated
 | trivy.vulnType | string | `nil` | vulnType can be used to tell Trivy to filter vulnerabilities by a pkg-type (library, os) |
 | trivyOperator.additionalReportLabels | string | `""` | additionalReportLabels comma-separated representation of the labels which the user wants the scanner pods to be labeled with. Example: `foo=bar,env=stage` will labeled the reports with the labels `foo: bar` and `env: stage` |
 | trivyOperator.configAuditReportsPlugin | string | `"Trivy"` | configAuditReportsPlugin the name of the plugin that generates config audit reports. |
+| trivyOperator.exludeImages | string | `""` | exludeImages is a glob patterns for excluding images from scanning. Example: `acr.azurecr.io/*/*` will exclude images from azure container registry. |
```

Comment:
Thank you tackling this issue!

There seems to be a consistent typo throughout this PR though: ˋexludeImagesˋ should probably be called ˋexcludeImagesˋ?

Reply:
@ybasket thanks for the catch up